lib/Configuration.js

```diff
@@ -102,8 +188,8 @@ export default class Configuration {
    */
   get(key, { fromFile } = {}) {
     return this.configs.find((config) => {
-      const appliesTo = config.applies_to || '**';
-      const appliesFrom = config.applies_from || '**';
+      const appliesTo = config.appliesTo || '**';
+      const appliesFrom = config.appliesFrom || '**';
```

Comment:
Oh, but we actually use the `config` object later. We can still simplify this a bit with some destructuring in the function body. I'll add a commit.